README.md

```diff
@@ -285,11 +285,16 @@ Pull the minimal values file from the current repo:
 
 ```bash
 curl -O https://raw.githubusercontent.com/Alfresco/alfresco-dbp-deployment/master/charts/incubator/alfresco-dbp/minimal-values.yaml
-sed -i s/REPLACEME/$LOCALIP/g minimal-values.yaml
+sed -i "" s/REPLACEME/$LOCALIP/g minimal-values.yaml
 ```
 
 ### 11. Deploy the DBP
 
+```bash
+# Make sure you have the latest Helm charts locally
+helm update
```

Comment:
See https://github.com/Alfresco/alfresco-dbp-deployment/pull/108#commitcomment-33513479.  The command for updating the repos has changed. The command 'helm update' is deprecated in favor of 'helm repo update'